tests/strategies/test_deepspeed_strategy.py

```diff
@@ -1185,3 +1198,26 @@ def test_deepspeed_multi_save_same_filepath(tmpdir):
     ckpt_path = os.path.join(trainer.checkpoint_callback.dirpath, "last.ckpt")
     expected = ["latest", "zero_to_fp32.py", "checkpoint"]
     assert set(expected) == set(os.listdir(ckpt_path))
+
+
+@RunIf(min_gpus=2, standalone=True, deepspeed=True)
+@pytest.mark.skipif(not _DEEPSPEED_GREATER_EQUAL_0_6, reason="requires deepspeed >= 0.6")
+@pytest.mark.parametrize("stage", [1, 2, 3])
+def test_deepspeed_with_bfloat16_precision(tmpdir, stage):
+    """Test that deepspeed works with bfloat16 precision."""
+    model = BoringModel()
+    trainer = Trainer(
+        default_root_dir=tmpdir,
+        strategy=f"deepspeed_stage_{stage}",
+        accelerator="gpu",
+        devices=2,
```

Comment:
This test doesn't need 2 gpus, just one would be fine

Reply:
Do you think it's relevant that we test the different stages for the assertions we have here?

We need to keep standalone tests as minimal and fast as possible because they are the current bottleneck in CI

Reply:
oh I see. let's just test stage=3, since it's important.